src/main/msp/msp_build_info.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
```

Comment:
Can you please use the new licence file found in the root directory of the repo :)